### DISCUSSION TITLE
LTMEET FLOOD Questions

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#891] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
1-How do I turn off LTMEET Flood 
2-Can a school block it 
3-If a school can block it can I prevent it (If it requires a update to block it)

**Comment by AshtonDavies** [comment, marked as answer]
1: Run chrome://restart in the address bar.
2: I believe it is not really blockable by administrators because if an extension page is blocked, you could probably just use a different one. And they don't usually want to block chrome://extensions either so I think you're good. It has lasted many months so far with no action as far as I've noticed.
3: It most likely would be prevented with an update if so, but even in that case I do believe this particular extension bypassing strategy would call for a difficult rework in the way Chrome works to patch, and the devs generally prefer to patch things in simple ways, as seen in the past, just eliminating them in a simple block check.

**Reply by CGRmags** [reply]
@AshtonDavies Thanks for answering my questions so quickly and in-depth, Thanks for all the help you have given me. I will ask you next time I have more questions

**Reply by AshtonDavies** [reply]
> @AshtonDavies Thanks for answering my questions so quickly and in-depth, Thanks for all the help you have given me. I will ask you next time I have more questions

Excellent, glad I could help!

**Reply by funnypenguin100** [reply]
It's actually quite funny that you mention that administrators "don't usually want to block chrome://extensions" because the entire chrome://extensions URL tree was blocked today. I'm sure it will cause chaos at our school, since a lot of extensions are quite important in terms of utility, but the admins are desperate.

**Reply by AshtonDavies** [reply]
> It's actually quite funny that you mention that administrators "don't usually want to block chrome://extensions" because the entire chrome://extensions URL tree was blocked today. I'm sure it will cause chaos at our school, since a lot of extensions are quite important in terms of utility, but the admins are desperate.

Yeah, well, chrome://extensions doesn't have any reason to be unblocked either. It just feels messed up to have blocked though.
